tests/DCPS/DomainRange/DomainRangeTest.cpp

```diff
@@ -142,13 +142,12 @@ ACE_TMAIN(int argc, ACE_TCHAR *argv[])
 
     int num_readers = num_participants == 1 ? 1 : num_participants - 1;
 
-    OPENDDS_VECTOR(DDS::Subscriber_var) subscribers;
-    OPENDDS_VECTOR(DDS::Topic_var) topics;
-
     for (OPENDDS_VECTOR(DDS::DomainId_t)::const_iterator it = domains.begin(); it != domains.end(); ++it)
     {
       DDS::DomainId_t domain = *it;
 
+      DDS::DomainParticipant_var participants[num_readers];
```

Comment:
This is an error in Azure Pipelines.  `num_readers` is not a compile-time constant.